src/tooling/debugging/vscode-debugging.md

```diff
@@ -59,15 +58,22 @@ ESP32 doesn't have a built-in JTAG interface, so you have to connect an external
 
 ## ESP32-C3
 
-Older versions with **revision < 3** **don't** have built-in JTAG interface.
+The availability of built-in JTAG interface depends on the ESP32-C3 revision:
 
-ESP32-C3 with **revision 3** **does** have a built-in JTAG interface, and you don't have to connect an external device to be able to debug. To get the chip revision, run the `cargo espflash board-info` command.
+- Revisions older than 3 **don't** have built-in JTAG interface.
+- Revitions 3 (and newer) **do** have a built-in JTAG interface, and you don't have to connect an external device to be able to debug.
```

Comment:
If revision 3 is the latest, it might be better to remove `(and newer)`.

Reply:
Not sure which revision is the latest, but I think there will be new revisions of it, so it might be worth it to leave it.

Reply:
Looks like there is an empty bullet point following this like. Please delete it.